src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -107,28 +102,31 @@ export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
     actions.setStatus("submissionFailed")
   }
 
-  const commonEventProperties = {
-    auction_slug: sale.id,
-    artwork_slug: artwork.id,
-    sale_id: sale._id,
-    user_id: me.id,
-  }
-
   function trackConfirmBidFailed(bidderId: string, errors: string[]) {
-    tracking.trackEvent({
+    trackEvent({
       action_type: Schema.ActionType.ConfirmBidFailed,
       bidder_id: bidderId,
       error_messages: errors,
-      ...commonEventProperties,
     })
   }
 
-  function trackConfirmBidSuccess(positionId: string, bidderId: string) {
-    tracking.trackEvent({
+  function trackConfirmBidSuccess(
+    positionId: string,
+    bidderId: string,
+    selectedBidAmountCents: number
+  ) {
+    trackEvent({
       action_type: Schema.ActionType.ConfirmBidSubmitted,
       bidder_position_id: positionId,
       bidder_id: bidderId,
-      ...commonEventProperties,
+      order_id: bidderId,
+      products: [
+        {
+          product_id: artwork._id,
+          quantity: 1,
+          price: selectedBidAmountCents / 100,
```

Comment:
This jumps out as something that might not respect currencies with a different multiplier. Is it possible to pass the cents amount itself to the analytics call? (presumably any calculation or conversion could be done there).

Reply:
This was lifted from the corresponding Force code. Agreed that it seems not robust to other currencies, but then again, it’s been in prod on Force for a few months. I think @cavvia is best equipped to comment here.

Reply:
Thanks for catching that! This data is just used by our marketing partner Criteo right now I think... yes, it is probably affected by currency variations but it's not a high priority area to fix for us right now, as it has been like this for a long time.